### PR TITLE
grepcidr: Init at 2.0

### PR DESCRIPTION
**Pull request by fadenb**
###### Motivation for this change
grepcidr was not available on NixOS: http://www.pc-tools.net/unix/grepcidr/

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/search/grepcidr/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  version = "2.0";
+
+  src = fetchurl {
+    url = "http://www.pc-tools.net/files/unix/grepcidr-${version}.tar.gz";
+    sha256 = "1yzpa1nigmmp4hir6377hrkpp0z6jnxgccaw2jbqgydbglvnm231";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp grepcidr $out/bin/
```
Comment (dotlambda) [inline pkgs/applications/search/grepcidr/default.nix]: Please use the default `installPhase`. You'll probably have to set something like
```
installFlags = [ "PREFIX=$out" ];
```

File: pkgs/applications/search/grepcidr/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "grepcidr-${version}";
+  version = "2.0";
+
+  src = fetchurl {
+    url = "http://www.pc-tools.net/files/unix/grepcidr-${version}.tar.gz";
```
Comment (dotlambda) [inline pkgs/applications/search/grepcidr/default.nix]: `grepcidr-${version}` can be replaced by `${name}`

**Review by dotlambda — CHANGES_REQUESTED** [review]

File: pkgs/applications/search/grepcidr/default.nix (hunk 1 above)
Comment (Mic92) [reply pkgs/applications/search/grepcidr/default.nix]: checked the makefile. `installFlags = [ "PREFIX=$(out)" ];` should work.

**Comment by dotlambda** [comment]
@GrahamcOfBorg build grepcidr

File: pkgs/applications/search/grepcidr/default.nix [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    sha256 = "1yzpa1nigmmp4hir6377hrkpp0z6jnxgccaw2jbqgydbglvnm231";
+  };
+
+  installFlags = [ "PREFIX=$(out)" ];
+
+  meta = with stdenv.lib; {
+    description = "Filter IPv4 and IPv6 addresses matching CIDR patterns";
+    homepage = http://www.pc-tools.net/unix/grepcidr/;
+    license = licenses.gpl3;
+    platforms = platforms.unix;
```
Comment (dotlambda) [inline pkgs/applications/search/grepcidr/default.nix]: Did you test this on macOS?

**Review by dotlambda — APPROVED** [review]

File: pkgs/applications/search/grepcidr/default.nix (hunk 3 above)
Comment (fadenb) [reply pkgs/applications/search/grepcidr/default.nix]: @dotlambda yes, tested and working with:
 - system: `"x86_64-darwin"`
 - host os: `Darwin 16.7.0, macOS 10.12.6`
 - multi-user?: `yes`
 - sandbox: `no`

**Pull request merged**